org.eclipse.winery.topologymodeler.ui/lib/tslint.json

```diff
@@ -0,0 +1,130 @@
+{
+  "rulesDirectory": [
+    "node_modules/codelyzer"
+  ],
+  "rules": {
+    "arrow-return-shorthand": true,
```

Comment:
Does the extension of the root tslint file not work?

Reply:
This tslint file is very likely never used